saos-webapp/src/main/webapp/WEB-INF/view/details/judgmentDetailsMeta.jsp

```diff
@@ -0,0 +1,176 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+<%@ include file="/WEB-INF/view/common/taglibs.jsp" %>
+
+
+<spring:message code="judgmentDetails.meta.keywords" var="detailsMetaKeywords" />
+
+<%-- JudgmentType --%>
+<c:if test="${!empty judgment.judgmentType}" >
+    <c:set var="metaJudgmentType" >
+        <saos:enum value="${judgment.judgmentType}" ></saos:enum>
+    </c:set>
+    <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaJudgmentType}" />
+</c:if>
+
+<%-- CaseNumbers --%>
+<c:if test="${!empty judgment.caseNumbers}" >
+    
+    <c:set var="metaCaseNumbers" >
+        <saos:caseNumber items="${judgment.caseNumbers}"/>
+    </c:set>
+    <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaCaseNumbers}" />
+</c:if>
+
+<%-- CourtType --%>
+<c:if test="${!empty judgment.courtType}" >
+
+    <spring:message code="pl.edu.icm.saos.persistence.model.CourtType.${judgment.courtType}" var="metaCourtType" />
+    <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaCourtType}" />
+</c:if>
+
+<%-- CommonCourtJudgment properties --%>
+<c:if test="${judgment.courtType == 'COMMON'}" >
+
+	<%-- CommonCourt --%>
+    <c:if test="${!empty judgment.courtDivision.court.name}" >
+        <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${judgment.courtDivision.court.name}" />
+	</c:if>
+    
+	<%-- CommonCourtDivision --%>
+    <c:if test="${!empty judgment.courtDivision.name}" >   
+        <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${judgment.courtDivision.name}" />
+    </c:if>
+
+</c:if>	
+
+<%-- SupremeCourtJudgment properties --%>
+<c:if test="${judgment.courtType == 'SUPREME'}" >
+
+    <c:if test="${!empty judgment.scChambers}" >
+        <c:set var="metaScChambers" value="" />
+        <c:forEach items="${judgment.scChambers}" var="chamber">
+            <c:set var="metaScChambers" value="${metaScChambers}, ${chamber.name }" />
+        </c:forEach>
+        
+        <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${metaScChambers}" />
+    </c:if>
+
+    <c:if test="${!empty judgment.scChamberDivision.name}" >
+        <c:set var="detailsMetaKeywords" value="${detailsMetaKeywords}, ${judgment.scChamberDivision.name}" />
+    </c:if>
+
```

Comment:
I've checked it and I got something like this: Postanowienie o sygnaturze II PK 144/09 wydane przez , Izba Pracy, Ubezpieczeń Społecznych i Spraw Publicznych w składzie sędziowskim Zbigniew Hajn.
There is a comma after przed. Also, I think it should start with 'Postanowienie SN...'